src/main/java/seedu/address/logic/commands/GetAttendanceByTgCommand.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIAL_GROUP;
+
+import java.util.List;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.student.Student;
+import seedu.address.model.student.TutorialGroup;
+import seedu.address.ui.AttendanceWindow;
+
+/**
+ * Retrieves the attendance of all students in a specific tutorial group.
+ */
+public class GetAttendanceByTgCommand extends Command {
+    public static final String COMMAND_WORD = "getattg";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Retrieves the attendance of all students in a specific tutorial group.\n"
+            + "Parameters: "
+            + PREFIX_TUTORIAL_GROUP + "TUTORIAL_GROUP\n"
+            + "Example: "
+            + COMMAND_WORD + " "
+            + PREFIX_TUTORIAL_GROUP + " A01";
+
+    public static final String MESSAGE_SUCCESS = "Attendance for students in tutorial group %1$s:\n%2$s";
+    public static final String MESSAGE_NO_STUDENTS = "No students found in tutorial group %1$s.";
```

Comment:
This might be just me but it feels like the message is focusing more on students not being found than the tutorial group probably being wrong which is why it's empty.